gson/src/main/java/com/google/gson/internal/reflect/ReflectionHelper.java

```diff
@@ -0,0 +1,71 @@
+package com.google.gson.internal.reflect;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Field;
+
+import com.google.gson.JsonIOException;
+
+public class ReflectionHelper {
+  private ReflectionHelper() { }
+
+  /**
+   * Tries making the field accessible, wrapping any thrown exception in a
+   * {@link JsonIOException} with descriptive message.
+   *
+   * @param field
```

Comment:
Have changed it. It probably depends on the IDE you are using, but I kind of like this wrapped documentation style because it makes the distinction between parameter name and description clearer.
Though it does not follow the style guide of many projects.